src/main/kotlin/com/idan/pokemon_hub/controller/PokemonController.kt

```diff
@@ -2,13 +2,32 @@ package com.idan.pokemon_hub.controller
 
 import com.idan.pokemon_hub.model.Pokemon
 import com.idan.pokemon_hub.service.PokemonService
+import org.springframework.http.HttpMethod
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.*
 
 @RestController
 @RequestMapping("/pokemons")
 class PokemonController(private val pokemonService: PokemonService) {
 
+    @RequestMapping(value = ["/"], method = [RequestMethod.OPTIONS])
+    fun optionsAll(): ResponseEntity<String> {
```

Comment:
You don't need to do this. You shouldn't do it.
Spring will take care of it on it's own when properly configured, that's what it is for.